housekeeping-vacuum-tool/pom.xml

```diff
@@ -95,6 +95,7 @@
               <artifactSet>
                 <includes>
                   <include>com.hotels:housekeeping-vacuum-tool</include>
+                  <include>com.hotels:hcommon-hive-metastore</include>
```

Comment:
Need a changelog entry for this, saying that this is now shaded into the tool and removed from the lib folder.